spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/eventhub/DefaultEventHubClientFactory.java

```diff
@@ -50,8 +51,7 @@
     private final EventHubNamespace namespace;
     private String checkpointStorageConnectionString;
 
-    public DefaultEventHubClientFactory(AzureAdmin azureAdmin, String namespace) {
-        Assert.notNull(azureAdmin, "azureAdmin can't be null.");
+    public DefaultEventHubClientFactory(@NonNull AzureAdmin azureAdmin, String namespace) {
```

Comment:
suggest to use `@NonNull` from `org.springframework.lang.NonNull`.

Reply:
What's the diff between lombok nonnull and spring?

Reply:
for stability consideration, some lombok may conflict with some maven plugin. If there are two similar annotation, I prefer the one from springframework.